oversweet-domain/src/main/java/com/depromeet/oversweet/domain/record/repository/FindRecordsRepositoryImpl.java

```diff
@@ -39,4 +39,15 @@ public List<RecordEntity> findRecordsByLocalDateTime(final Long memberId, final
                 .fetch();
 
     }
+
+    @Override
+    @Transactional(readOnly = true)
+    public RecordEntity findRecordById(Long id) {
+        return queryFactory.selectFrom(recordEntity)
+                .join(recordEntity.drink, drinkEntity).fetchJoin()
+                .join(recordEntity.drink.franchise, franchiseEntity).fetchJoin()
+                .where(recordEntity.member.id.eq(id))
+                .fetchOne();
+    }
 }
```

Comment:
Optional 처리는 필요가 없을까요??

```suggestion
    @Override
@Transactional(readOnly = true)
public Optional<RecordEntity> findRecordById(Long id) {
    RecordEntity recordEntity = queryFactory.selectFrom(QRecordEntity.recordEntity)
            .join(QRecordEntity.recordEntity.drink, QDrinkEntity.drinkEntity).fetchJoin()
            .join(QRecordEntity.recordEntity.drink.franchise, QFranchiseEntity.franchiseEntity).fetchJoin()
            .where(QRecordEntity.recordEntity.member.id.eq(id))
            .fetchOne();

    return Optional.ofNullable(recordEntity);
}

```

Reply:
넵넵 당장은 비즈니스 로직에 사용하지 않지만 추후 활용을 위해 Optional 처리를 해주는 것이 좋을 것 같습니다.
수정하겠습니다!